src/filters/UserFilters.jsx

```diff
@@ -35,6 +35,7 @@ export default class UserFilters extends React.Component {
     }
   }
   updateUserList() {
+    this.props.dispatch(clearUserList());
```

Comment:
I was thinking it should be here since this component controls new searches, and doesn't want to be explicitly aware of infinite scroll components that might be its children.